src/creatwth.cpp

```diff
@@ -499,37 +508,57 @@ static BOOL UpdateFrom32To64(HANDLE hProcess, HMODULE hModule, WORD machine,
 }
 #endif // DETOURS_64BIT
 
-typedef BOOL(WINAPI *LPFN_ISWOW64PROCESS)(HANDLE, PBOOL);
+namespace Detour {
```

Comment:
>Detour [](start = 10, length = 6)

Detours? 

Also why does this need a name space, these should all be static functions?

Reply:
The latest commit has resolved it.